k8s-daemonset/k8s/linkerd-grpc.yml

```diff
@@ -28,10 +28,9 @@ data:
       label: outgoing
       experimental: true
       dtab: |
-        /srv                      => /#/io.l5d.k8s/default/grpc;
-        /svc/helloworld.Hello     => /srv/hello;
-        /svc/helloworld.World     => /srv/world;
-        /srv/world                => /srv/world-v1;
+        /srv       => /#/io.l5d.k8s/default/grpc;
+        /svc       => /$/io.buoyant.http.domainToPathPfx/srv;
+        /srv/World => /srv/world-v1;
```

Comment:
`/srv => /srv` type rules are a bit dangerous since they can lead to loops if you're not careful.  Could do:

```
/srv => /#/io.l5d.k8s/... ; 
/grpc => /srv ;
/grpc/World => srv/world-v1 ;
/svc => /$/.../grpc ;
```